### PR TITLE
tka: add public API on NodeKeySignature for key information

### PR DESCRIPTION
**Pull request by twitchyliquid64**
This is needed in the coordination server.

### CODE REVIEW

File: tka/sig.go [hunk 1]
```diff
@@ -96,6 +96,15 @@ type NodeKeySignature struct {
 	WrappingPubkey []byte `cbor:"6,keyasint,omitempty"`
 }
 
+// UnverifiedWrappingPublic returns the public key which must sign a
+// signature which embeds this one, if any.
```
Comment (maisem) [inline tka/sig.go:100]: this needs more explanation

Comment (twitchyliquid64) [reply tka/sig.go:100]: Ive added a line pointing the reader towards documentation about wrapping keys.

File: tka/sig.go [hunk 2]
```diff
@@ -96,6 +96,18 @@ type NodeKeySignature struct {
...
+//
+// See docs on NodeKeySignature.WrappingPubkey & SigRotation for documentation
+// about wrapping public keys.
+//
+// SAFETY: The caller MUST verify the signature using
+// Authority.NodeKeyAuthorized if treating this as authentic information.
```
Comment (maisem) [inline tka/sig.go:106]: why is it `Unverified`?

Comment (twitchyliquid64) [reply tka/sig.go:106]: Because its possible for the caller to get at this information without verifying the signature.

**Review by maisem — APPROVED** [review]

File: tka/sig.go (hunk 2 above)
Comment (maisem) [reply tka/sig.go:106]: thanks!

**Pull request merged**